lcls_tools/common/measurements/emittance_measurement.py

```diff
@@ -0,0 +1,79 @@
+from lcls_tools.common.devices.reader import create_magnet, create_screen, create_wire
+from lcls_tools.common.devices.magnet import MagnetCollection
+from lcls_tools.common.devices.device import Device
+from lcls_tools.common.measurements.measurement import Measurement
+
+from meme.model import Model
+
+class QuadScanEmittance(Measurement):
+    model_beamline: str
+    magnet_area: str
+    magnet_name: str
+    scan_values: list[float]
+    rmats: list
+    _model: Model = None
+    _magnet_collection: MagnetCollection = None
+    _magnet_settings: list[dict] = None
+
+    to_device_area: str
```

Comment:
instead of `to_device` lets do `measurement_device: Device`. Also will need to define a pydantic object that defines the beamline properties that we need to do the quad scan, ie. quad length, rmats, design twiss values etc. We can define a helper function that uses meme to populate these values